modules/tinymce/src/plugins/lists/main/ts/core/Bookmark.ts

```diff
@@ -12,17 +12,6 @@ interface Bookmark {
   endOffset?: number;
 }
 
-/**
- * Returns a range bookmark. This will convert indexed bookmarks into temporary span elements with
- * index 0 so that they can be restored properly after the DOM has been modified. Text bookmarks will not have spans
- * added to them since they can be restored after a dom operation.
- *
- * So this: <p><b>|</b><b>|</b></p>
- * becomes: <p><b><span data-mce-type="bookmark">|</span></b><b data-mce-type="bookmark">|</span></b></p>
- *
- * @param  {DOMRange} rng DOM Range to get bookmark on.
- * @return {Object} Bookmark object.
- */
```

Comment:
Why was this deleted? It still seems useful 🤔

Reply:
Because moxidoc would fail. Though it might have been enough to remove the param-return and the rest can be kept.